CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+* (improvement) [#13254](https://github.com/cosmos/cosmos-sdk/pull/13268) Change `I` suffix to prefix in the relevant Golang interfaces
```

Comment:
Can you move that under the `API Breaking Changes` section (l.107)?

Reply:
Done. Note that I was following the instructions in `https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#pull-requests` and figured someone (in the future) is responsible from moving it from `Unreleased` to `API Breaking`.